### PR TITLE
Implement the fix

### LINKED ISSUE #1897: Command modal should open with first command expanded
When opening a command modal, the first command should be expanded by default. This will save the user having to click to open and is useful when the modal is opened in response to a command in progress.

## Current behavior 

Screen Shot 2020-05-14 at 7 45 27 AM

## Desired behavior

Screen Shot 2020-05-14 at 7 47 54 AM

### PR DESCRIPTION
**Pull request by nlinker**
out

Fixes #1897 
Also added a small space before the "Cancel" button.

Signed-off-by: Nick Linker <nlinker@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2134)

### CODE REVIEW

File: iml-gui/crate/src/components/command_modal.rs [hunk 1]
```diff
@@ -422,7 +425,7 @@ fn job_item_combine(parent: Node<Msg>, acc: Vec<Node<Msg>>, _ctx: &mut Context)
 fn job_item_cancel_button(job: &Arc<RichJob>, cancelling: bool) -> Node<Msg> {
     if let Some(trans) = find_cancel_transition(job) {
         let cancel_btn: Node<Msg> = div![
-            class![C.inline, C.px_1, C.rounded_lg, C.text_white, C.cursor_pointer],
+            class![C.inline, C.ml_1, C.px_1, C.rounded_lg, C.text_white, C.cursor_pointer],
```
Comment (nlinker) [inline iml-gui/crate/src/components/command_modal.rs:428]: The indent for the "Cancel" button.

File: iml-gui/crate/src/components/command_modal.rs [hunk 2]
```diff
@@ -150,6 +150,9 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg, GMsg>)
 
             match cmds {
                 Input::Commands(cmds) => {
+                    if let Some(cmd) = cmds.first() {
+                        model.select.perform_click(TypedId::Cmd(cmd.id));
+                    }
```
Comment (nlinker) [inline iml-gui/crate/src/components/command_modal.rs:155]: `select` will have the first command id, and then `FetchTree` will do the rest

**Comment by johnsonw** [comment]
Looks like firefox tests are failing. Has this been rebased on master recently?

**Comment by nlinker** [comment]
> Looks like firefox tests are failing. Has this been rebased on master recently?

Let's try this.

**Review by jgrund — APPROVED** [review]

**Pull request merged**